run.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+
+source /home/ec2-user/.env.development
 cd /home/ec2-user/nina-indexer
-pm2 stop nina-indexer && pm2 delete nina-indexer
-pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer
-pm2 stop nina-api && pm2 delete nina-api
-pm2 start yarn --interpreter bash --name nina-api -- start:api
\ No newline at end of file
+pm2 start /home/ec2-user/.yarn/bin/yarn --interpreter bash --name nina-indexer -- start:indexer --no-autorestart
```

Comment:
wondering what will happen in the case that the server needs to be restarted - will we have to do it manually because of `--no-autorestart`?

Reply:
ah we can actually remove this now it's actually a no-op in this case.